docs-chef-io/content/inspec/resources/parse_config_file.md

```diff
@@ -145,3 +126,35 @@ Use `standalone_comments: false`, to parse the following:
 
     'key = value # comment'
     params['key'] = 'value'
+
+## Matchers
+
+For a full list of available matchers, please visit our [matchers page](/inspec/matchers/).
+
+## Examples
+
+The following examples show how to use this Chef InSpec audit resource.
+
+### Test a configuration setting
+
+    describe parse_config_file('/path/to/file.conf') do
+     its('PARAM_X') { should eq 'Y' }
+    end
+
+### Use options, and then test a configuration setting
+
+    describe parse_config_file('/path/to/file.conf', { multiple_values: true }) do
+     its('PARAM_X') { should include 'Y' }
+    end
+
+### Test a file with an ini-like structure (such as a yum.conf)
+
+    describe parse_config_file('/path/to/yum.conf') do
+     its('main') { should include('gpgcheck' => '1') }
+    end
+
+### Test a configuration setting containing dots
```

Comment:
@rgeissert When referring to "dots," are you talking about periods (`.`) within a URL, a specific technical file, or something else?

Reply:
@mjingle It should read: "Test a configuration setting whose name contains dots" (or periods, or full stops, as you prefer :)). As in: `author.name = Raphael` in a configuration file.

Reply:
@rgeissert Awesome! Thank you for confirming. That will help inform my suggested edits to you, and then we'll get this docs part finished.